rpc/backend/backend.go

```diff
@@ -115,7 +115,7 @@ type EVMBackend interface {
 
 	// Tx Info
 	GetTransactionByHash(txHash common.Hash) (*rpctypes.RPCTransaction, error)
-	GetTxByEthHash(txHash common.Hash) (*ethermint.TxResult, error)
+	GetTxByEthHash(txHash common.Hash) (*ethermint.TxResult, bool, error)
```

Comment:
How does this mesh with rpc responses over http requests? it seems like each function returns one data field, and one error. does adding a third return val mess it up?

Reply:
doesn't really matter as there's a wrapper for the actual eth rpcs and this function is just called there and we handle the change.